xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -50,6 +50,8 @@ class CRenderManager;
 class IRenderMsg
 {
   friend CRenderManager;
+public:
+  virtual ~IRenderMsg() = default;
```

Comment:
Do note that this is only relevant if you are (potentially) going to destroy the class via a pointer to the interface.